engine/src/cubos/engine/gizmos/plugin.cpp

```diff
@@ -183,6 +203,9 @@ static void drawGizmosSystem(Write<Gizmos> gizmos, Write<GizmosRenderer> gizmosR
 {
     auto screenSize = (*window)->framebufferSize();
 
+    gizmosRenderer->renderDevice->setFramebuffer(gizmosRenderer->framebuffer);
```

Comment:
I would change the name of the `framebuffer` field to `idFramebuffer` or something like that. This way its a bit confusing